com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -1123,9 +1124,11 @@ RegId AddBricks(List<Brick> bricks, ProbeBrickPool.DataLocation dataloc, ProbeBr
                 }
             }
 
-            // Update the pool and index and ignore any potential frame latency related issues for now
+            // Update textures with incoming SH data and index and ignore any potential frame latency related issues for now
```

Comment:
The comment seems less precise now. 

What textures? We have intermediate textures. Also without commas now it looks like we are updating textures with index data. 

Maybe
> Update pool textures with incoming SH data and ignore any potential frame latency related issues for now. 

And move the index part straight above the m_Index.AddBricks

Reply:
Agreed (not even sure why I changed that tbh)